### PR TITLE
Fix intermittent failures with freetext and stamp tests

### PR DESCRIPTION
**Pull request by calixteman**
They're potentially due to some concurrent access to the system clipboard. So this patch makes them sequential.

### CODE REVIEW

**Review by timvandermeij — APPROVED** [review]
Nice, I was just about to make this exact patch today after coming to the same conclusion about concurrent system clipboard access yesterday. This was a really tough one to figure out, but eventually it worked out by copying text from another window and then seeing that text from the system clipboard being pasted in the browser. The chance of that happening is also pretty high given that the Chrome and Firefox tests run pretty much at the same time, and it also explains why sometimes seemingly "random" results were seen in the assertions.

In short, thank you for doing this, and with this we should finally have gotten rid of all those pesky copy/paste intermittents and `waitForEvent` logs!
